### DISCUSSION TITLE
Using action-runner-controller with Openshift

### DISCUSSION THREAD

**Discussion by metabsd** [actions/actions-runner-controller#1961] [Questions]
Hello, I tried to use action-runner-controller in one of our Openshift cluster. I could not find any mention about using this project with Openshift. After reviewing the logs I think I have understood that you need to enable anyuid for everything to work as expected. I would like to know if others have already succeeded with Openshift without releasing the security.

Thank you!

**Comment by dav-pascual** [comment]
Hi @metabsd ! Did you get this to work with Openshift?

**Reply by metabsd** [reply]
Yes, someone here make it work. You need to know how ?

**Reply by dav-pascual** [reply]
Yes, if you would be so kind :) 
I'm trying to make it work without modifying openshift anyuid permissions.

Looking at this thread it looks hopeless though https://github.com/actions/actions-runner-controller/discussions/2703
